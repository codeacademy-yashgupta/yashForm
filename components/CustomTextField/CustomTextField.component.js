import React, { Component } from 'react';
import { TextInput, View, StyleSheet } from 'react-native';
import style from './CustomTextField.style';

const styles = StyleSheet.create(style);
export default class CustomTextField extends Component {
  render() {
    return (
      <View>
        <TextInput
          onChange={this.props.onChange}
          style={styles.textInput}
        />
      </View>
    );
  }
}
