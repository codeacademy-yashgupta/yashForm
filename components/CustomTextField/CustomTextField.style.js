export default {
  textInput: {
    borderBottomWidth: 1, borderBottomColor: 'black', margin: 20, fontSize: 22,
  },
};
