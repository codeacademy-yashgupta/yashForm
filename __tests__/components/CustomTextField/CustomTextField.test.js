import React from 'react';
import renderer from 'react-test-renderer';
import CustomTextField from '../../../components/CustomTextField/CustomTextField.component';
import 'react-native';

describe('CustomTextField', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<CustomTextField />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
