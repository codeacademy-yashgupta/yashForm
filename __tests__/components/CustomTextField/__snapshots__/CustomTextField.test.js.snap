// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomTextField should render correctly 1`] = `
<View>
  <TextInput
    allowFontScaling={true}
    style={
      Object {
        "borderBottomColor": "black",
        "borderBottomWidth": 1,
        "fontSize": 22,
        "margin": 20,
      }
    }
    underlineColorAndroid="transparent"
  />
</View>
`;
